web/src/pages/settings/ManageLibrariesModal.tsx

```diff
@@ -97,6 +97,7 @@ const EditLibraryForm: React.FC<{ library: LibraryDetails; onDone: () => void }>
         submitLabel="Save"
       >
         <TextInput required label="Name:" value={name} onChange={setName} id="name" />
+        <TextInput tabIndex={-1} required label="Folder:" value={library.rootFolder} id="folder" />
```

Comment:
Should we default `tabIndex` to -1 for readonly fields? Or would there be a reason to have a readonly field you _can_ tab-navigate to?

Reply:
Nothing gets past you. Done.

Reply:
Uhmm, I think you did it the wrong way around ;P

```
tabIndex={onChange && -1}
```

means you can _only_ tab to readonly fields :sweat_smile:

Reply:
heheh, oops

Reply:
Fixed